feat: support `NODE_EXTRA_CA_CERTS` (#41689)
electron/electron: docs/api/environment-variables.md

--http-parser
```

### `NODE_EXTRA_CA_CERTS`

See [Node.js cli documentation](https://github.com/nodejs/node/blob/main/doc/api/cli.md#node_extra_ca_certsfile) for details.

```sh
export NODE_EXTRA_CA_CERTS=/path/to/cert.pem 

> ```

Reviewer: We should possibly mention the fuses here and in the NODE_OPTIONS section above?

Author: done in 94303959a3

--http-parser
```

> The [`nodeOptions` fuse](../tutorial/fuses.md#L27) need to be enabled in order to use this variable.

Reviewer: That fuse is enabled by default, so this should probably be
```suggestion
If the [`nodeOptions` fuse](../tutorial/fuses.md#L27) is disabled, `NODE_OPTIONS` will be ignored.
```

Same for the others below.

### `NODE_EXTRA_CA_CERTS`

See [Node.js cli documentation](https://github.com/nodejs/node/blob/main/doc/api/cli.md#node_extra_ca_certsfile) for details.

```sh
export NODE_EXTRA_CA_CERTS=/path/to/cert.pem 
```

> If the [`nodeOptions` fuse](../tutorial/fuses.md#L27) is disabled, `NODE_OPTIONS` will be ignored.

Reviewer 2: ```suggestion
If the [`nodeOptions` fuse](../tutorial/fuses.md#L27) is disabled, `NODE_EXTRA_CA_CERTS` will be ignored.
```
